.circleci/build.sh

```diff
@@ -6,18 +6,17 @@ set -e
 # defaults to packages for minimal intervention in the ci config
 FOLDER="${FOLDER:-packages}"
 
-DOCKER_IMAGE_TAG=$DOCKER_IMAGE_TAG-$SPECKLE_SERVER_PACKAGE
+DOCKER_IMAGE_TAG=speckle/speckle-$SPECKLE_SERVER_PACKAGE
 
 IMAGE_VERSION_TAG=$(./.circleci/get_version.sh)
 
 docker build --build-arg SPECKLE_SERVER_VERSION=$IMAGE_VERSION_TAG -t $DOCKER_IMAGE_TAG:latest . -f $FOLDER/$SPECKLE_SERVER_PACKAGE/Dockerfile
-docker tag $DOCKER_IMAGE_TAG:latest $DOCKER_IMAGE_TAG:$IMAGE_VERSION_TAG
 
-echo "$DOCKER_REG_PASS" | docker login -u "$DOCKER_REG_USER" --password-stdin $DOCKER_REG_URL
-docker push $DOCKER_IMAGE_TAG:latest
-docker push $DOCKER_IMAGE_TAG:$IMAGE_VERSION_TAG
+docker tag $DOCKER_IMAGE_TAG:latest $DOCKER_IMAGE_TAG:$IMAGE_VERSION_TAG
 
 if [[ "$IMAGE_VERSION_TAG" =~ ^[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
-  docker tag $DOCKER_IMAGE_TAG:latest $DOCKER_IMAGE_TAG:2
-  docker push $DOCKER_IMAGE_TAG:2
+  docker tag $DOCKER_IMAGE_TAG:2
```

Comment:
missing parameter to docker tag

Reply:
its actually tagget on line 15. if we're pushing an alpha tag or a release tag, its value is stored on the $IMAGE_VERSION_TAG and applied above. The only tag, that we're adding only when we're releasing is the `:2` tag.

Below with the `docker push -a` pushes all tags that is attached to the given image

Reply:
fixed it